app/src/main/kotlin/no/nav/k9punsj/innsending/InnsendingClient.kt

```diff
@@ -70,9 +71,9 @@ interface InnsendingClient {
         ).keyValue
     }
 
-    fun sendKopierJournalpost(info: KopierJournalpostInfo) = send(mapKopierJournalpost(info))
+    fun sendKopierJournalpost(info: KopierJournalpostInfo) = runBlocking { send(mapKopierJournalpost(info))  }
```

Comment:
Hvorfor ikke suspende funkskonen her også, istendenfor å bruke runBlocking?

Reply:
Sant, å her trenger jag jo å implementere støtte for kopiering av journalpost 😭 Jobber vidare med det..